action.yml

```
@@ -18,15 +18,19 @@ inputs:
     description: Set to true in order to enable timestamps on log messages
     required: false
     default: 'true'
-  # TODO(user): Specify your inputs here
-  input-name:
-    description: Some input description
+  aws-account-id:
+    description: AWS account ID to use for ECR repo
     required: true
-
-# TODO(user): Define your outputs here
-outputs:
-  some-output:
-    description: 'Some output description'
+  ecr-repo-name:
+    description: AWS ECR Repo Name to use to view scan
+    required: true
+  ecr-repo-tag:
+    description: AWS ECR Repo image tag to use to view scan
+    required: true
+  use-alpha:
+    description: AWS ECR Repo Alpha image
+    required: true
+    default: 'true'
```

Set the `required` to false so that we don't have to populate the input each time
```suggestion
    required: false
    default: 'true'
```

---

updated to suggestion